medicines/search-client/src/models.rs

```diff
@@ -171,14 +173,14 @@ impl From<IndexResult> for IndexEntry {
                 Some(k) => k,
                 None => "".to_owned(),
             },
-            territory: match res.territory {
-                Some(t) => t,
-                None => "".to_owned(),
-            },
             title: res.title,
             pl_number: vec![],
             file_name: res.file_name,
             doc_type: res.doc_type,
+            territory: match res.territory {
+                Some(t) => t,
+                None => TerritoryType::UK,
```

Comment:
I can't remember how the `IndexEntry` is used - what's the implication of defaulting to UK here instead of making it optional on `IndexEntry` and defaulting to None?

Reply:
Good shout here, will send this through as an `Option`.

Reply:
Found a few other places where this was used as a fixed value rather than parsed from incoming XML and have addressed this in the latest commit, so real good spot with this 👍 .